.idea/gradle.xml

```diff
@@ -5,19 +5,14 @@
       <GradleProjectSettings>
         <option name="distributionType" value="LOCAL" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleHome" value="$APPLICATION_HOME_DIR$/gradle/gradle-2.14.1" />
+        <option name="gradleHome" value="C:\Program Files (x86)\Android\gradle\gradle-2.14.1" />
```

Comment:
This should be reverted back to using the `$APPLICATION_HOME_DIR$` variable so the project can be run on non-Windows operating systems.